### PR TITLE
Updating eventing-contrib 0.9.0 images

### PR DESCRIPTION
**Pull request by alanconway**
Added images to core-services/image-mirroring/knative/mapping_knative_v0_9_quay.
Removed some stale images from contrib-release-v0.9.0.yaml files.

Signed-off-by: Alan Conway <aconway@redhat.com>

### CODE REVIEW

**Comment by matzew** [comment]
/hold

this looks wrong

File: core-services/image-mirroring/knative/mapping_knative_v0_9_quay [hunk 1]
```diff
@@ -17,4 +17,5 @@ registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-ingress
...
 registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-sendevent quay.io/openshift-knative/knative-eventing-sendevent:v0.9.0
 registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-webhook quay.io/openshift-knative/knative-eventing-webhook:v0.9.0
-registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-sources-controller quay.io/openshift-knative/knative-eventing-sources-controller:v0.9.0
\ No newline at end of file
+registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-sources-controller quay.io/openshift-knative/knative-eventing-sources-controller:v0.9.0
+registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-sources-kafka-source-adapter kafka-source-controller kafka-channel-controller kafka-channel-dispatcher kafka-channel-webhook camel-source-controller github-receive-adapter github-source-controller quay.io/openshift-knative/knative-eventing-sources-kafka-source-adapter kafka-source-controller kafka-channel-controller kafka-channel-dispatcher kafka-channel-webhook camel-source-controller github-receive-adapter github-source-controller:v0.9.0
```
Comment (matzew) [inline core-services/image-mirroring/knative/mapping_knative_v0_9_quay]: this misses the channel for kafka ...

File: ci-operator/config/openshift/knative-eventing-contrib/openshift-knative-eventing-contrib-release-v0.9.0__variant.yaml [hunk 2]
```diff
@@ -85,22 +77,6 @@ images:
       - destination_dir: .
         source_path: /go/bin/camel-source-controller
   to: knative-eventing-sources-camel-source-controller
-- dockerfile_path: openshift/ci-operator/knative-images/websocketsource/Dockerfile
```
Comment (matzew) [inline ci-operator/config/openshift/knative-eventing-contrib/openshift-knative-eventing-contrib-release-v0.9.0__variant.yaml]: I see these images still here:

https://github.com/knative/eventing-contrib/tree/v0.9.0/cmd

and in ours:

https://github.com/openshift/knative-eventing-contrib/tree/release-v0.9.0/cmd

**Comment by alanconway** [comment]
I think it's fixed up now. I was getting confused between master and 0.9.0 branches and running make from master, which gives wrong results.

**Comment by matzew** [comment]
@alanconway the force push from d8de911 to 50b8685 still lacks some images

**Comment by alanconway** [comment]
Hah! I think I have it this time.

File: core-services/image-mirroring/knative/mapping_knative_v0_9_quay [hunk 3]
```diff
@@ -17,4 +17,20 @@ registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-ingress
...
+registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-sources-kafka-channel-webhook quay.io/openshift-knative/knative-eventing-sources-kafka-channel-webhook:v0.9.0
+registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-sources-camel-source-controller quay.io/openshift-knative/knative-eventing-sources-camel-source-controller:v0.9.0
+registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-sources-github-receive-adapter quay.io/openshift-knative/knative-eventing-sources-github-receive-adapter:v0.9.0
+registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-sources-github-source-controller quay.io/openshift-knative/knative-eventing-sources-github-source-controller:v0.9.0
+registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-sources-./cmd/heartbeats quay.io/openshift-knative/knative-eventing-sources-./cmd/heartbeats:v0.9.0
+registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-sources-./cmd/websocketsource quay.io/openshift-knative/knative-eventing-sources-./cmd/websocketsource:v0.9.0
```
Comment (matzew) [inline core-services/image-mirroring/knative/mapping_knative_v0_9_quay]: ` knative-eventing-sources-./cmd/websocketsource  `

that's wrong

File: core-services/image-mirroring/knative/mapping_knative_v0_9_quay [hunk 4]
```diff
@@ -17,4 +17,16 @@ registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-ingress
...
+registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-sources-camel-source-controller quay.io/openshift-knative/knative-eventing-sources-camel-source-controller:v0.9.0
+registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-sources-github-receive-adapter quay.io/openshift-knative/knative-eventing-sources-github-receive-adapter:v0.9.0
+registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-sources-github-source-controller quay.io/openshift-knative/knative-eventing-sources-github-source-controller:v0.9.0
+registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-sources-heartbeats quay.io/openshift-knative/knative-eventing-sources-heartbeats:v0.9.0
+registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-sources-websocketsource quay.io/openshift-knative/knative-eventing-sources-websocketsource:v0.9.0
+registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-sources-event_display quay.io/openshift-knative/knative-eventing-sources-event_display:v0.9.0
```
Comment (matzew) [inline core-services/image-mirroring/knative/mapping_knative_v0_9_quay]: that image does not exist.  CI server requires that we do not have `_` in the name ( no idea why).

When we generate the CI config, we already "catch" that, see:
https://github.com/openshift/knative-eventing/blob/master/openshift/ci-operator/generate-ci-config.sh#L50

@alanconway I guess that's needed when generating this file too :-) 

see that `docker pull registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-sources-event-display` works

Comment (alanconway) [reply core-services/image-mirroring/knative/mapping_knative_v0_9_quay]: Fixed, is this the final touch?

**Pull request merged**